bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/nifs/NifsAo.scala

```diff
@@ -61,7 +61,7 @@ case class NifsAo(blueprint: SpNifsBlueprintAo, exampleTarget: Option[SPTarget])
 
   // ### Target Group
   // INCLUDE {1},{2},ACQ,SCI,{7},{8} in target-specific Scheduling Group
-  include(List(1, 2) ++ acq ++ List(sci, 7, 8): _*) in TargetGroup
+  include(List(0, 1) ++ acq ++ List(sci, 7, 8): _*) in TargetGroup
```

Comment:
Why was the change from `List(1, 2)` to `List(0, 1)` as in the spec it has the text **INCLUDE {1}, {2}**?
It is just a matter of the comment on the line above being off?